modules/logging-loki-gui-install.adoc

```diff
@@ -35,14 +33,132 @@ include::snippets/logging-stable-updates-snip.adoc[]
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging # <2>
+spec:
+  size: 1x.small # <3>
+  storage:
+    schemas:
+    - version: v12
+      effectiveDate: "2022-06-01"
+    secret:
+      name: logging-loki-s3 # <4>
+      type: s3 # <5>
+      credentialMode: # <6>
+  storageClassName: <storage_class_name> # <7>
+  tenants:
+    mode: openshift-logging # <8>
+----
+<1> Use the name `logging-loki`.
+<2> You must specify the `openshift-logging` namespace.
+<3> Specify the deployment size. In the {logging} 5.8 and later versions, the supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`.
+<4> Specify the name of your log store secret.
+<5> Specify the corresponding storage type.
+<6> Optional field, logging 5.9 and later. Supported user configured values are as follows: static is the default authentication mode available for all supported object storage types using credentials stored in a Secret. token for short-lived tokens retrieved from a credential source. In this mode the static configuration does not contain credentials needed for the object storage. Instead, they are generated during runtime using a service, which allows for shorter-lived credentials and much more granular control. This authentication mode is not supported for all object storage types. token-cco is the default value when Loki is running on managed STS mode and using CCO on STS/WIF clusters.
+<7> Specify the name of a storage class for temporary storage. For best performance, specify a storage class that allocates block storage. Available storage classes for your cluster can be listed by using the `oc get storageclasses` command.
+<8> LokiStack defaults to running in multi-tenant mode, which cannot be modified. One tenant is provided for each log type: audit, infrastructure, and application logs. This enables access control for individual users and user groups to different log streams.
+--
++
+[IMPORTANT]
+====
+It is not possible to change the number `1x` for the deployment size.
+====
+
+. Click *Create*.
+
+. Create an OpenShift Logging instance:
+
+.. Switch to the *Administration* -> *Custom Resource Definitions* page.
+
+.. On the *Custom Resource Definitions* page, click *ClusterLogging*.
+
+.. On the *Custom Resource Definition details* page, select *View Instances* from the *Actions* menu.
+
+.. On the *ClusterLoggings* page, click *Create ClusterLogging*.
++
+You might have to refresh the page to load the data.
+
+.. In the YAML field, replace the code with the following:
++
+--
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: instance # <1>
+  namespace: openshift-logging # <2>
+spec:
+  collection:
+    type: vector
+  logStore:
+    lokistack:
+      name: logging-loki
+    retentionPolicy:
+      application:
+        maxAge: 7d
+      audit:
+        maxAge: 7d
+      infra:
+        maxAge: 7d
```

Comment:
This is not valid for Loki. The CLO doesn't manage retention for Loki, it's managed by the Loki CR as described in https://docs.openshift.com/container-platform/4.17/observability/logging/log_storage/cluster-logging-loki.html#logging-loki-retention_cluster-logging-loki.